star_tides/core/actions/get_contact_action.py

```diff
@@ -0,0 +1,15 @@
+'''star_tides.core.action.get_contact_action
+'''
+from star_tides.services.databases.mongo.models.contact_model import ContactModel
+from star_tides.core.actions.base_action import Action
+
+class GetContactAction(Action):
+    '''
+    GetContactAction fetches the Contact from the database using the given ID.
+    '''
+    def __init__(self, contact_id: str) -> None:
+        super().__init__()
+        self.contact_id = contact_id
+
+    def run(self):
+        return ContactModel.objects.get(id=self.contact_id)
```

Comment:
You should dump here! By dumping here we make it so other controllers and actions can call this action repeatedly without us worrying about side effects of modifying the model in other places.